src/Service/Ses/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: Added support for `getSuppressedDestination` and `deleteSuppressedDestination` to manage SESv2 suppression list entries.
```

Comment:
```suggestion
- Added support for `getSuppressedDestination` and `deleteSuppressedDestination` to manage SESv2 suppression list entries.
```

the `AWS api-change: ` is used by the bot when it updated automatically the API with the latest AWS specs